modules/gke-nodepool/variables.tf

```diff
@@ -40,6 +40,18 @@ variable "initial_node_count" {
   default     = 1
 }
 
+variable "kubelet_config" {
+  description = "Kubelet configuration."
+  type        = map(string)
```

Comment:
Would it make sense to use an object type, to ensure there are no typos?

```
type = object({
  cpu_cfs_quota = string
  cpu_cfs_quota_period = string
  cpu_manager_policy = string
})
```

Reply:
Only those 3 are [supported](https://cloud.google.com/sdk/gcloud/reference/beta/container/node-pools/create#--system-config-from-file), so probably a good idea. And same thing for sysctls.

Reply:
It makes all sense, of course. Uploaded fix.